src/harness/fake_db_server.py

```diff
@@ -0,0 +1,78 @@
+#    Copyright 2016 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+"""A fake implementation of database.
+
+A local test server could be run by using "python fake_db_server.py".
+
+"""
+
+import json
+import threading
+from BaseHTTPServer import HTTPServer, BaseHTTPRequestHandler
+
+class FakeDatabaseTestHarness(threading.Thread):
+
+  def __init__(self, url = None, database_file = None):
+    """Test Harness acting as a Http Server to receive Pull/GET
+    Requests from SAS Under Test
+    parameters: is the dict variable for database.
```

Comment:
As commented in other PRs, please pass parameters directly rather than using a dictionary. This will help us avoid errors and it makes the code much more readable.

Reply:
Passing   the parameters( port, host_name, path, databaseFile) as individual variables   to fake database server.